File: cristopher/Problema0112/problema0112.java
package problemasProgramacao.cristopher.Problema0112;

import java.util.Scanner;

public class problema0112 {

    public static void main(String[] args) {
        
        Scanner input = new Scanner(System.in);

        System.out.println("Informe um número qualquer: ");
        float numero = Float.valueOf(input.nextLine());

        if(numero > 0){
            System.out.println("Imprimindo a raiz quadrada: " + Math.sqrt(numero));

        }else{
            System.out.println("Imprimindo o quadrado do número: " + Math.pow(numero, 2));
        }



        input.close();
    }
    
}

File: cristopher/Problema0112/problema0112.py
import math


numero = float(input("Informe um número qualquer: "))

if numero > 0:
    print(f"Imprimindo a raiz do número: {math.sqrt(numero)}")
else:
    print(f"Imprimindo o quadrado do número: {math.pow(numero, 2)}")